wagtail/contrib/settings/models.py

```diff
@@ -144,7 +145,10 @@ def for_site(cls, site):
         return instance
 
     def __str__(self):
-        return "%s for %s" % (self._meta.verbose_name.capitalize(), self.site)
+        return _("%(site_setting)s for %(site)s") % {
+            "site_setting": self._meta.verbose_name.capitalize(),
```

Comment:
as per discussion on https://github.com/wagtail/wagtail/issues/9219

let's pull out this `capitalize` call and either a. replace with capfirst OR find the location in the template and update with `|capfirst` filter (ideal if not too complex).